Sources/DeclarationHelpers.swift

```diff
@@ -1095,6 +1095,9 @@ extension Formatter {
         _ category: Category
     ) -> Bool {
         switch category.type {
+        case .swiftUIPropertyWrapper:
+            return true
+
         case .instanceProperty:
             return true
```

Comment:
We could simplify this to:

```suggestion
        case .instanceProperty, .swiftUIPropertyWrapper:
            return true
```